workflows/smrnaseq.nf

```diff
@@ -7,6 +7,7 @@ include { CAT_FASTQ                        } from '../modules/nf-core/cat/fastq/
 include { CONTAMINANT_FILTER               } from '../subworkflows/local/contaminant_filter'
 include { FASTQC                           } from '../modules/nf-core/fastqc/main'
 include { FASTQ_FASTQC_UMITOOLS_FASTP      } from '../subworkflows/nf-core/fastq_fastqc_umitools_fastp'
+include { FASTP3                           } from '../modules/local/trim3p.nf'
```

Comment:
yes, I agree, I talked to @apeltzer. I tried to use the module twice, but I couldn't get it work since it is using some symlinks internally and it is overriding the current files:

```
[ ! -f  ${prefix}.fastq.gz ] && ln -sf $reads ${prefix}.fastq.gz
```

Did you try the solution you proposed? I think changing the names of the files in between could be an option, but not sure.

I told @apeltzer that we can have this until somebody figure out. We can leave an issue open with help wanted or something like that.

Reply:
@nschcolnicov / @atrigila and myself just discussed it again and it should work the way @nschcolnicov just mentioned and also supplying the `stageInMode: 'copy'` in the `modules.config` for the `FASTP3` then.